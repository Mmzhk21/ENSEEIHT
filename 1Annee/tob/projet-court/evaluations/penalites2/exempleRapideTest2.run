java allumettes.Jouer Xavier@humain Ordinateur@rapide <<EOF
1
1
3
EOF

