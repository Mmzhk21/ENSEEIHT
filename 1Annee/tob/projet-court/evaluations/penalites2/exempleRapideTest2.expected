Nombre d'allumettes restantes : 13
Xavier, combien d'allumettes ? Xavier prend 1 allumette.

Nombre d'allumettes restantes : 12
Ordinateur prend 3 allumettes.

Nombre d'allumettes restantes : 9
Xavier, combien d'allumettes ? Xavier prend 1 allumette.

Nombre d'allumettes restantes : 8
Ordinateur prend 3 allumettes.

Nombre d'allumettes restantes : 5
Xavier, combien d'allumettes ? Xavier prend 3 allumettes.

Nombre d'allumettes restantes : 2
Ordinateur prend 2 allumettes.

Ordinateur perd !
Xavier gagne !
